eth/tracers/api.go

```diff
@@ -741,9 +741,9 @@ func containsTx(block *types.Block, hash common.Hash) bool {
 // TraceTransaction returns the structured logs created during the execution of EVM
 // and returns them as a JSON object.
 func (api *API) TraceTransaction(ctx context.Context, hash common.Hash, config *TraceConfig) (interface{}, error) {
-	tx, blockHash, blockNumber, index, err := api.backend.GetTransaction(ctx, hash)
-	if err != nil {
-		return nil, err
+	tx, blockHash, blockNumber, index, _ := api.backend.GetTransaction(ctx, hash)
```

Comment:
always catch errors

Reply:
I am not sure if we should return the err we can print it instead also there a `tx == nil` check after it

Reply:
but before it was err != nil

Reply:
BTW if you look to the implementation of `GetTransaction` you will find err is always `nil`